src/core_write.cpp

```diff
@@ -225,9 +245,17 @@ void TxToUniv(const CTransaction& tx, const uint256& hashBlock, UniValue& entry,
         ScriptPubKeyToUniv(txout.scriptPubKey, o, true);
         out.pushKV("scriptPubKey", o);
         vout.push_back(out);
+
+        amt_total_out += txout.nValue;
     }
     entry.pushKV("vout", vout);
 
+    if (txundo != nullptr && !tx.IsCoinBase()) {
```

Comment:
Should be enough `if (txundo) {`?

Reply:
Need to remove the assert below too, since fees will be negative on a generation transaction.

Reply:
Also might need to subtract subsidy amount from generation.

Reply:
Nevermind, I see that by initialising txundo to nullptr, this is safe.